<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTip</class>
 <widget class="QDialog" name="MyTip">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTip</string>
  </property>
  <property name="styleSheet">
   <string notr="true">image: url(:/QtGuiApplication/Resources/tip_background.jpg);</string>
  </property>
  <widget class="QPushButton" name="tipCancel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tipOK">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mohu_station">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Adobe Arabic&quot;;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../QtGuiApplication.qrc"/>
 </resources>
 <connections/>
</ui>
